Makefile

```diff
@@ -13,8 +13,8 @@ check-go:
 	@which go > /dev/null || (echo "Go is not installed. Please install Go." && exit 1)
 
 install-lint-deps:
-	@which golangci-lint > /dev/null || (echo "Required golangci-lint not found. Installing it..." && GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@latest)
-	@which staticcheck > /dev/null || (echo "Required staticcheck not found. Installing it..." && GO111MODULE=on go get honnef.co/go/tools/cmd/staticcheck)
+	@which golangci-lint > /dev/null || (echo "Required golangci-lint not found. Installing it..." && go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest)
+	@which staticcheck > /dev/null || (echo "Required staticcheck not found. Installing it..." && go install honnef.co/go/tools/cmd/staticcheck@latest)
```

Comment:
we _might_ want to lock this but keeping the local version on `latest` does make it a bit more likely that we'll keep up with upstream changes